infrastructure/azure.tf

```diff
@@ -374,3 +374,51 @@ resource "azurerm_eventhub_authorization_rule" "azurerm_apim_eventhub_rule" {
     send                = true
     manage              = false
 }
+
+# Azure Container Service (Kubernetes)
+
+resource "azurerm_container_service" "azure_container_service-1" {
+  name                   = "aks-1"
+  location               = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name    = "${azurerm_resource_group.azurerm_resource_group.name}"
+  orchestration_platform = "Kubernetes"
+
+  master_profile {
+    count      = 3 // 3 or 5 for HA
+    dns_prefix = "aks-1-master"
+  }
+
+  linux_profile {
+    admin_username = "admin"
+
+    ssh_key {
+      key_data = "TODO" // TODO
+    }
+  }
+
+  agent_pool_profile {
+    name       = "default"
+    count      = 3
+    dns_prefix = "aks-1-agent"
+    vm_size    = "Standard_A0" // TODO
+  }
+
+  service_principal {
+    client_id     = "00000000-0000-0000-0000-000000000000" // TODO
+    client_secret = "00000000000000000000000000000000" // TODO
+  }
+
+  diagnostics_profile {
+    enabled = false
+  }
+
+  tags {
+    environment = "${var.environment}"
+  }
+}
+
+
+// TODO public IPs for the k8s cluster https://www.terraform.io/docs/providers/azurerm/r/public_ip.html
+
+// TODO LB for routing public traffic to the k8s cluster https://www.terraform.io/docs/providers/azurerm/r/loadbalancer.html
```

Comment:
we may be able to create an external LB from k8s, linked to a fixed public IP, unfortunately the [azure doc](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/container-service/kubernetes/container-service-tutorial-kubernetes-deploy-cluster.md) is very poor, need some experimentation